### PR TITLE
Apply contribution guidelines to "Cpf" rule

### PR DESCRIPTION
**Pull request by williamespindola**

### CODE REVIEW

File: library/Rules/Cpf.php [hunk 1]
```diff
@@ -13,8 +13,19 @@
 
 namespace Respect\Validation\Rules;
 
-class Cpf extends AbstractRule
+/**
+ * @author Alexandre Gaigalas <alexandre@gaigalas.net>
```
Comment (batusa) [inline library/Rules/Cpf.php]: Could you add `Validates a Brazillian CPF number.` as a description for this class?

Comment (williamespindola) [reply library/Rules/Cpf.php]: for sure

Comment (williamespindola) [reply library/Rules/Cpf.php]: And fix the name of Alexandre too.

File: tests/integration/rules/cpf.phpt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+use Respect\Validation\Validator as v;
+
+try {
+    v::cpf()->check('this thing');
+} catch (CpfException $exception) {
+    echo $exception->getMessage().PHP_EOL;
+}
+
+try {
+    v::cpf()->check('68786265440');
```
Comment (henriquemoody) [inline tests/integration/rules/cpf.phpt]: The goal of the integration tests is to just test how the exceptions behave. That said we only need to test `check()`, `assert()`, `not(check())`, and `not(assert()` when they throw exception.

Could you remove some extra tests and put them to the unit test instead if they are not there already?

File: tests/unit/Rules/CpfTest.php [hunk 3]
```diff
@@ -13,123 +13,69 @@
 
 namespace Respect\Validation\Rules;
 
-use PHPUnit\Framework\TestCase;
+use Respect\Validation\Test\RuleTestCase;
 
 /**
  * @group  rule
```
Comment (henriquemoody) [inline tests/unit/Rules/CpfTest.php]: Can you remove the extra space?

File: tests/unit/Rules/CpfTest.php [hunk 4]
```diff
@@ -13,123 +13,69 @@
...
+ *
  * @covers \Respect\Validation\Rules\Cpf
  * @covers \Respect\Validation\Exceptions\CpfException
```
Comment (henriquemoody) [inline tests/unit/Rules/CpfTest.php]: 🔪, we are testing the exceptions in the PHPT files

File: tests/unit/Rules/CpfTest.php [hunk 5]
```diff
@@ -13,123 +13,69 @@
...
+ *
+ * @author Alexandre Gaigalas <alexandre@gaigalas.net>
+ * @author Gabriel Caruso <carusogabriel34@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
+ * @author Jair Henrique <jair.henrique@gmail.com>
+ * @author Jean Pimentel <jeanfap@gmail.com>
+ * @author William Espindola Yet <oi@williamespindola.com.br>
```
Comment (henriquemoody) [inline tests/unit/Rules/CpfTest.php]:
```diff
-- @author William Espindola Yet <oi@williamespindola.com.br>
++ @author William Espindola <oi@williamespindola.com.br>
```

File: tests/unit/Rules/CpfTest.php [hunk 6]
```diff
@@ -13,123 +13,69 @@
...
-use PHPUnit\Framework\TestCase;
+use Respect\Validation\Test\RuleTestCase;
 
 /**
  * @group  rule
+ *
  * @covers \Respect\Validation\Rules\Cpf
  * @covers \Respect\Validation\Exceptions\CpfException
+ *
+ * @author Alexandre Gaigalas <alexandre@gaigalas.net>
```
Comment (henriquemoody) [inline tests/unit/Rules/CpfTest.php]:
```diff
-- Alexandre Gaigalas <alexandre@gaigalas.net>
++ Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
```

**Comment by henriquemoody** [comment]
Merged by #985.

Great to see your contributions! 🐼

**Pull request closed without merging**